bundles/AdminBundle/src/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1723,47 +1739,53 @@ public function bulkExportAction(Request $request): JsonResponse
             ];
         }
 
-        $classes = new DataObject\ClassDefinition\Listing();
-        $classes->setOrder('ASC');
-        $classes->setOrderKey('id');
-        $classes = $classes->load();
+        if($this->getAdminUser()->isAllowed('classes')) {
```

Comment:
Why would we need this check?
Admin User always has all permissions?

What about the Fieldcollections above? Why would the check not be there but for classes and object bricks?

Reply:
? `$this->getAdminUser()` has nothing to do with admin users. It just returns the `\Pimcore\Model\User` object of the currently logged user - and this can or cannot have the permission `classes`.